lvms-operator/lvmcluster.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: crc-lvmcluster
+  namespace: openshift-storage
+spec:
+  storage:
+    deviceClasses:
+    - name: rhel
```

Comment:
why is this called `rhel`? Maybe topolvm or such?
https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.12/html-single/deploying_and_managing_logical_volume_manager_storage_on_single_node_openshift_clusters/index#scaling-storage-of-single-node-openshift-cluster-using-rhacm_sno calls it `vg1`